### PR TITLE
CEDS-2343 - convert 'Seals' pages to GDS

### PR DESCRIPTION
**Pull request by SteveSugden**

### CODE REVIEW

File: conf/messages.en [hunk 1]
```diff
@@ -1158,18 +1158,17 @@ supplementary.itemsSummary.noOfPackages = Number of packages
...
+declaration.seal.question = What is the security seal?
+declaration.seal.summary.heading = Security seals
+declaration.seal.summary.noSeals = No seals
+declaration.seal.add.first = Does container {0} have any security seals?
+declaration.seal.add.another = Do you need to add another security seal?
+declaration.seal.remove.title = Are you sure you want to remove this security seal for container {0}?
+declaration.seal.remove.heading = Security seal
+declaration.seal.remove.hint = Remove security seal {0}
+declaration.transport.sealId.empty.error = Enter a seal ID
+declaration.transport.sealId.error.invalid = Seal ID must not be more than 20 alpha numeric characters
```
Comment (PatrykRudnicki) [inline conf/messages.en:1171]: 👍

File: app/views/declaration/seal_remove.scala.html [hunk 2]
```diff
@@ -17,56 +17,92 @@
...
-                </thead>
-                <tbody>
-                <tr>
-                    <td >@sealId</td>
-                </tr>
-                </tbody>
-            </table>
-        </div>
+    @formHelper(action = SealController.submitSealRemove(mode, containerId, sealId), 'autoComplete -> "off") {
+        @errorSummary(form.errors)
```
Comment (iservport) [inline app/views/declaration/seal_remove.scala.html:89]: Don't we need  `@helper.CSRF.formField` here?

File: app/views/declaration/seal_summary.scala.html [hunk 3]
```diff
@@ -17,64 +17,113 @@
...
-                    @if(seals.nonEmpty) {
-                        @components.fields.removable_elements_table(
-                            title = Some(messages("standard.seal.summary.heading")),
-                            elements = seals.map(seal => LabelledValue(seal.id)),
-                            removeButtonClass = "button-link",
-                            removeButtonHint = "standard.seal.remove.hint"
-                        )
-                    }
+    @formHelper(action = SealController.submitSummaryAction(mode, containerId), 'autoComplete -> "off") {
+        @errorSummary(form.errors)
```
Comment (iservport) [inline app/views/declaration/seal_summary.scala.html:111]: Same as above with `@helper.CSRF.formField`

File: app/views/declaration/seal_remove.scala.html (hunk 2 above)
Comment (PatrykRudnicki) [reply app/views/declaration/seal_remove.scala.html:89]: FormHelper contains the CSRF token inside

File: app/views/declaration/seal_add.scala.html [hunk 4]
```diff
@@ -20,36 +20,51 @@
...
-) {
-
-    @components.back_link(controllers.declaration.routes.SealController.displaySealSummary(mode, containerId), mode)
+@govukLayout(
+    title = Title("declaration.seal.title", "declaration.seal.summary.heading", containerId),
+    backButton = Some(BackButton(messages("site.back"), controllers.declaration.routes.SealController.displaySealSummary(mode, containerId)))) {
 
-    @components.error_summary(form.errors)
+    @formHelper(action = SealController.submitAddSeal(mode, containerId), 'autoComplete -> "off") {
+        @helper.CSRF.formField
```
Comment (iservport) [inline app/views/declaration/seal_add.scala.html:52]: So we don't need CSRF here

Comment (PatrykRudnicki) [reply app/views/declaration/seal_add.scala.html:52]: As Mauricio pointed, this CSRF formFiled is not necessary

Comment (PauloPortugal) [reply app/views/declaration/seal_add.scala.html:52]: `formHelper: uk.gov.hmrc.play.views.html.helpers.FormWithCSRF`

**Pull request merged**